.config/dotnet-tools.json

```diff
@@ -7,6 +7,12 @@
       "commands": [
         "dotnet-cake"
       ]
+    },
+    "swashbuckle.aspnetcore.cli": {

```

Comment:
Why?

I add this to `BSN.IpTables.Api/.config`

Reply:
"Why?"
Because we use swagger cli in post-build of IpTables.CLI, so we need to restore it before that.

"I add this to BSN.IpTables.Api/.config"
Not clear what you are saying. You mean you added it in the last commit? No, it didn't happen, see the code.

Reply:
Discussed in [Gharar](http://www.gharar.ir).